traffic_ops/testing/api/v14/withobjs_test.go

```diff
@@ -79,6 +80,7 @@ var withFuncs = map[TCObj]TCObjFuncs{
 	CacheGroups:                          {CreateTestCacheGroups, DeleteTestCacheGroups},
 	CacheGroupsDeliveryServices:          {CreateTestCachegroupsDeliveryServices, DeleteTestCachegroupsDeliveryServices},
 	CacheGroupParameters:                 {CreateTestCacheGroupParameters, DeleteTestCacheGroupParameters},
+	Capabilities:                         {CreateTestCapabilities, DeleteTestCapabilities},
```

Comment:
no DeleteTestCapabilities causing build failures

Reply:
Fixed. That was harder than I thought - turns out API capabilities aren't managed by WithObjs and some new tests that have been added since this PR opened were broken because of that.